tests/index/sorted_index_tests.cpp

```diff
@@ -1787,6 +1787,102 @@ TEST_P(SortedIndexTestCase, doc_removal_same_key_within_trx) {
   }
 }
 
+bool Insert(irs::IndexWriter::Transaction& ctx, const tests::document* d) {
+  auto doc = ctx.Insert();
+  if (d->sorted && !doc.Insert<irs::Action::STORE_SORTED>(*d->sorted)) {
+    return false;
+  }
+  return doc.Insert<irs::Action::INDEX>(d->indexed.begin(), d->indexed.end()) &&
+         doc.Insert<irs::Action::STORE>(d->stored.begin(), d->stored.end());
+}
+
+TEST_P(SortedIndexTestCase, doc_removal_same_key_within_trx_flush) {
```

Comment:
This test failed for https://github.com/iresearch-toolkit/iresearch/commit/48b8fd9cbab5303c232642b36c3f8e43bcec28ff and current master